frontend/src/components/EditCurrentSchedule/ScheduleBoard/ScheduleBoardItem.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import { MdGroup } from 'react-icons/md';
+
+import { IoMdMore } from 'react-icons/all';
+
+import Card from '@material-ui/core/Card';
+import {
+    FORM_GROUPED_LABEL,
+    FORM_HOURS_LABEL,
+} from '../../../constants/translationLabels/formElements';
+import {
+    COMMON_EDIT,
+    COMMON_DELETE_HOVER_TITLE,
+} from '../../../constants/translationLabels/common';
+import { actionType } from '../../../constants/actionTypes';
+import { getTeacherName } from '../../../helper/renderTeacher';
+import i18n from '../../../i18n';
+
+const ScheduleItem = (props) => {
+    const {
+        deleteScheduleItem,
+        checkRoomAvailability,
+        itemData,
+        getLessonsByGroupId,
+        selectByGroupId,
+        t,
+        openDialogWithData,
+    } = props;
+    const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
@HiroFive Could you use only useState without React?

Reply:
fixed